RTS/2.1-Tipping_Curve/red_tipping_curve.py

```diff
@@ -487,10 +487,12 @@ def plot_data_el(Tsys,Tant,title='',units='K',line=42,aperture_efficiency=None,f
     plt.ylabel('$T_{sys}/\eta_{ap}$  (K)')
     return fig
 
-def r_lim(dataf,func=np.min):
+def r_lim(dataf,func=np.nanmin):
     """ Returns the func of the data , not used on nans"""
     dataf = np.array(dataf)
     index = ~np.isnan(dataf)
```

Comment:
It's always a bit tricky for me to read something like "all (not-nan is False)", trying to parse the Boolean logic of it all.

Rather call the blandly-named variable `index` `valid`, then you can say `if not valid.any():`.

Reply:
I have made the changes